Formula/osm2pgsql.rb

```diff
@@ -26,6 +25,12 @@ class Osm2pgsql < Formula
 
   uses_from_macos "expat"
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5" # C++17
+
```

Comment:
I don't think we should add any more Linux-only GCC dependencies. This formula has only 11 installs on Linux in the past year; we can drop the Linux bottle.